components/aws/sagemaker/tests/integration_tests/resources/config/ack-hosting-update/config.yaml

```diff
@@ -0,0 +1,26 @@
+PipelineDefinition: resources/definition/update_hosting_v2_pipeline.py
+TestName: ack-update-endpoint-test
+Timeout: 3600
+ExpectedPrediction:
+  predictions:
+    - distance_to_cluster: 7.448746204376221
+      closest_cluster: 2.0
+ExpectedEndpointConfig: ept2
+Arguments:
+  region: ((REGION))
```

Comment:
I am unfamiliar with this syntax, is this required by one of the tools we are using or is this a personal choice

Reply:
Yea its required by the tools